Doc/c-api/type.rst

```diff
@@ -217,6 +219,19 @@ The following functions and structs are used to create
       The function now accepts a single class as the *bases* argument and
       ``NULL`` as the ``tp_doc`` slot.
 
+.. c:function:: PyObject* PyType_FromMetaModuleAndSpec(PyTypeObject *meta, PyObject *module, PyType_Spec *spec, PyObject *bases)
```

Comment:
I don't think that it's worth it to mention every parameter in the function name, I propose a shorter name:

```suggestion
.. c:function:: PyObject* PyType_FromMetaclass(PyTypeObject *metaclass, PyObject *module, PyType_Spec *spec, PyObject *bases)
```

Note: I prefer to use "metaclass" for the parameter name in the public doc, but I'm fine with keeping "meta" in the C code ;-)

Reply:
I can change it in both.